tests/index.test.ts

```diff
@@ -35,9 +36,22 @@ describe("fontawesomeSubset", () => {
 
         const tempDir = await createTempDir();
         await fontawesomeSubset(subsets, tempDir, { package: PACKAGE });
+        const fontFileExtension = detectFontFileExtension(
+            require.resolve(`@fortawesome/fontawesome-${PACKAGE}`)
+        );
 
         for (const expectation of expected) {
-            const font = loadSync(`${tempDir}${SEP}${expectation.family}.ttf`);
+            let font;
+            const fontFilePath = `${tempDir}${SEP}${expectation.family}.${fontFileExtension}`;
+
+            if (fontFileExtension === "ttf") {
+                font = loadSync(fontFilePath);
+            } else {
+                font = parse(
+                    Uint8Array.from(await wawoff2.decompress(readFileSync(fontFilePath))).buffer
+                );
+            }
```

Comment:
Similarly I think this can be reduced to

```ts
const font = parse(
    Uint8Array.from(await wawoff2.decompress(readFileSync(
        `${tempDir}${SEP}${expectation.family}.woff2`
    ))).buffer
);
```

Looks like we'll still need the wawoff dependency until opentype releases their new version that has native woff2 support.